integreat_cms/cms/migrations/0025_update_roles.py

```diff
@@ -34,7 +34,7 @@ def update_roles(apps, schema_editor):
 
     # Assign the correct permissions
     for role_name in dict(roles.CHOICES):
-        group = Group.objects.filter(name=role_name).first()
+        group, _ = Group.objects.get_or_create(name=role_name)
```

Comment:
Ah, stupid me that I have not thought of this case when creating those migrations. Still not feeling completely comfortable with changing the past migrations, but in this case I think the production server should be fine.